tasks/18_humanize-format/humanize-format.js

```diff
@@ -0,0 +1,7 @@
+export function humanizeFormat(num) {
+  if (typeof num === 'undefined') return '';
+  if (num % 10 === 1) return num + 'st';
+  else if (num % 10 === 2) return num + 'nd';
+  else if (num % 10 === 3) return num + 'rd';
+  else return num + 'th';
```

Comment:
Again not cover with tests, but what will passing 11, 12 and 13 as params to your function return?

Not necessarily, but this can be covered using switch statement too.

Reply:
I Added the logic for 11,12,13, 111,112... and also refactored the code to use the `switch` statement.